doc/platforms.md

```diff
@@ -199,6 +200,25 @@ Google Scholar does not support adding specific pieces of content.
 
 Google Scholar does not support adding XPOC URI on the account page.
 
+## Rumble
+
+### Manifest content
+
+-   properties shared by `accounts` and `content`:
+    -   `platform`: "Rumble"
+    -   `account`: the owner's channel name `<accountname>`
+-   `accounts` properties:
+    -   `url`: `https://rumble.com/c/<accountname>/`
```

Comment:
Both YouTube and Rumble display the account's bio in the /about/ page. For YouTube, we say that should be the account URL. Perhaps we should do the same here, so validation tool can match the account URL with the location of the XPOC URI.